library/CMakeLists.txt

```diff
@@ -83,6 +83,7 @@ list(APPEND src_crypto ${thirdparty_src})
 set(src_x509
     certs.c
     pkcs11.c
+    pkcs7.c
```

Comment:
`MBEDTLS_PKCS7_USE_C` needs to be added to the exclusion list in `include_in_crypto` in `scripts/config.py`.